src/js/reducers/purchaseOrderReducer.jsx

```diff
@@ -1,4 +1,4 @@
-import { FETCH_PURCHASE_ORDER_STATUSES } from 'actions/types';
+import { FETCH_PAYMENT_TERMS, FETCH_PURCHASE_ORDER_STATUSES } from 'actions/types';
 
 const initialState = {
   statuses: [],
```

Comment:
you haven't added an `initialState` for `paymentTerms`. I hope it won't break anything and won't cause any null pointers

Reply:
Thanks for spotting that, I'll make sure to add it in one of the other PR's tat I'l be working on